src/components/SearchResultsList/SearchResultsList.js

```diff
@@ -85,7 +89,7 @@ const SearchResultsList = ({
   const recordToHighlight = useHighlightEditedRecord(authorities);
 
   const columnMapping = {
-    [searchResultListColumns.SELECT]: null,
+    [searchResultListColumns.SELECT]: <Checkbox onChange={() => toggleSelectAll()} checked={selectAll} title={selectAll ? 'Unselect all records on this page' : 'Select all records on this page'} />,
```

Comment:
Please format it with each prop on a new line
Also, we shouldn't have text hardcoded in component, we should add translations to en.json so the text can be translated into different languages.
```
[searchResultListColumns.SELECT]: (
  <Checkbox
    onChange={() => toggleSelectAll()}
    checked={selectAll}
    title={selectAll ? intl.formatMessage({ id: ... }) : intl.formatMessage({ id: ... })
  />
)
```